indico/client/client.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, config: IndicoConfig = None):
             urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
         self.config = config
         self._http = HTTPClient(config)
+        self.ipa_version = self._http.execute_request(GraphQLRequest("query getIPAVersion {\n  ipaVersion\n}\n"))['ipaVersion']
```

Comment:
Wondering now if this should be a method vs. attribute... curious for your thoughts Meg

Reply:
method is good 👍